api/test/wfl/integration/modules/covid_test.clj

```diff
@@ -346,29 +347,30 @@
 (def ^:private submission-id-mock (str (UUID/randomUUID)))
 
 (def ^:private running-workflow-mock
-  {:status           "Running"
-   :workflowId       (str (UUID/randomUUID))
-   :entityName "entity"
-   :inputResolutions [{:inputName (str method-name-mock ".input")
-                       :value     "value"}]})
+  {:entityName   "entity"
+   :id           (str (UUID/randomUUID))
+   :inputs       {:input "value"}
+   :status       "Running"
+   :workflowName fake-method-name})
 
 (def ^:private succeeded-workflow-mock
-  {:status         "Succeeded"
-   :workflowId     (str (UUID/randomUUID))
-   :entityName     "entity"
-   :inputResolutions [{:inputName (str method-name-mock ".input")
-                       :value     "value"}]})
+  {:entityName   "entity"
+   :id           (str (UUID/randomUUID))
+   :inputs       {:input "value"}
+   :status       "Succeeded"
+   :workflowName fake-method-name})
 
 ;; when we create submissions, workflows have been queued for execution
 (defn ^:private mock-firecloud-create-submission [& _]
-  (let [enqueue #(-> % (dissoc :workflowId) (assoc :staus "Queued"))]
+  (let [enqueue #(-> % (dissoc :id) (assoc :staus "Queued"))]
```

Comment:
Would be nice to centralize these key changes somehow.

Reply:
The trouble is this spans two data models. We should tidy this up but I'm not going to for a late fix